portfolio/src/main/webapp/style.css

```diff
@@ -1,35 +1,46 @@
 .container-fluid {
+  background-color: #fffaf5;
   font-family: Calibri, Candara, Segoe, 'Segoe UI', Optima, Arial, sans-serif;
 }
- 
+
 .scrollable {
   height: 600px;
   overflow: scroll;
 }
- 
+
 #calendar-wrapper,
 #new-event-wrapper,
 #new-task-wrapper,
 #schedule-result-wrapper {
-  background-color: #fffaf5;
-  border-radius: 50px;
   padding: 2%;
 }
- 
+
 #nav-bar {
   background-color: #dae8fe;
   width: 100%;
 }
- 
-#new-event-list-wrapper {
-  width: 80%;
+
+.btn-primary {
```

Comment:
This style seems to general. I think you want to be more specific. Otherwise all primary buttons will get extra margin top.

Reply:
Done.